server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts

```diff
@@ -41,7 +41,7 @@ export class RiddlerResources implements IResources {
 export class RiddlerResourcesFactory implements IResourcesFactory<RiddlerResources> {
     public async create(config: Provider): Promise<RiddlerResources> {
         // Database connection
-        const mongoUrl = config.get("mongo:endpoint") as string;
+        const mongoUrl = config.get("mongo:operationsDbEndpoint") as string;
```

Comment:
Riddler is supposed to use the globalDB right?

Reply:
Sorry just catching up to this, was OOF sick for a couple days. Yes, riddler will eventually use the globalDB - I plan to do that in a separate PR.